### ISSUE TITLE
Allow End key in Combat log to scroll to bottom

### ISSUE THREAD

**Issue by derickso** [factubsio/BubbleTweaks#2]
**Is your feature request related to a problem? Please describe.**
I'm frustrated when I scroll through the combat log but then the scroll bar is too small and there is no easy way to scroll to the bottom of the text area

**Describe the solution you'd like**
Allow the End key to function inside the combat log to scroll to the bottom

**Comment by Decair** [comment]
Agree, the combat log scroll bar sucks.  That said, there's a workaround to get to the bottom that I discovered the other day which may be useful to you: Change what you're filtering the log on (change from All to Combat) and it scrolls to the bottom).

I play in TBM, would love to have a better way to skip up to the start of the last person/creature's turn.  That's not that useful at level 1, but by the time you're in your teens, the start of the last person's turn can be 60+ rows up with smatterings of other team members making multiple opportunity attacks due to team feats, so it can take a while to parse.

**Comment by factubsio** [comment]
I'm working on an enhanced combat log with collapsible events for each characters turn.
